purpleair-aqi.js

```diff
@@ -459,11 +452,13 @@ function calculateLevel(aqi) {
 /**
  * Get the AQI trend
  *
- * @param {{ v1: number; v3: number; }} stats
+ * @param {{ pm2.5: number; pm2.5_10minute: number; }} stats
  * @returns {string}
  */
-function getAQITrend({ 'pm2.5': partLive, 'pm2.5_10minute': partTime }) {
-    const partDelta = partTime - partLive;
+function getAQITrend(stats) {
+  const partLive = stats["pm2.5"];
```

Comment:
TextMate was choking on the magic destructuring stuff, so I just moved it into the function.